packages/core/echo/echo-db/src/core-db/core-databasse.test.ts

```diff
@@ -6,53 +6,58 @@ import { effect } from '@preact/signals-core';
 import { expect } from 'chai';
 
 import { Trigger } from '@dxos/async';
-import { type DocHandle } from '@dxos/automerge/automerge-repo';
 import { createIdFromSpaceKey } from '@dxos/echo-pipeline';
 import { type SpaceDoc } from '@dxos/echo-protocol';
 import { create, type EchoReactiveObject, Expando } from '@dxos/echo-schema';
 import { registerSignalRuntime } from '@dxos/echo-signals';
 import { PublicKey } from '@dxos/keys';
-import { describe, test } from '@dxos/test';
+import { describe, openAndClose, test } from '@dxos/test';
 import { range } from '@dxos/util';
 
 import { getObjectCore } from './doc-accessor';
+import { type DocHandleClient } from '../client';
 import { createTestRootDoc, TestBuilder, TestPeer } from '../testing';
 
 describe('CoreDatabase', () => {
   describe('space fragmentation', () => {
-    const createSpaceFragmentationTestBuilder = () => new TestBuilder({ spaceFragmentationEnabled: true });
+    const createSpaceFragmentationTestBuilder = async () => {
+      const builder = new TestBuilder({ spaceFragmentationEnabled: true });
```

Comment:
unrelated but, I think space fragmentation should be made the default now?

Reply:
Yes, that is a legacy test builder, we have a new one, we need to tweak those tests for a new builder, I do not want to blow this PR even more